src/flask_app.py

```diff
@@ -47,4 +48,9 @@ def flask_environment(self) -> dict[str, str]:
         secret_key_env = f"{FLASK_ENV_CONFIG_PREFIX}SECRET_KEY"
         if secret_key_env not in env:
             env[secret_key_env] = self._charm_state.flask_secret_key
+        for proxy_variable in ("http_proxy", "https_proxy", "no_proxy"):
+            proxy_value = os.environ.get(f"JUJU_CHARM_{proxy_variable.upper()}")
```

Comment:
May I ask why the JUJU_CHARM_ prefix? It seems like the environment variable doesn't have the `JUJU_CHARM` prefix.

Reply:
I think this is defined by juju: https://juju.is/docs/sdk/charm-environment-variables